### ISSUE TITLE
Great job！When the dataset will be release?

### ISSUE THREAD

**Issue by BossZard** [uni-medical/SAM-Med3D#2]
Great job！When the dataset will be release :)

**Comment by blueyo0** [comment]
Hi, BossZard

Thank you for your recognition and attention :)

The specific date has not been determined yet due to some issues with licenses and copyrights. 
We need to further confirm the appropriate form for the data release. 

What we can confirm is that we will release the validation set first to facilitate verification by everyone (also because the validation set is smaller and has fewer conflicts). 

We will announce the details as soon as the data release is ready.

**Comment by blueyo0** [comment]
Hi, some important training datasets are listed here. We fine-tuned a new version of SAM-Med3D called SAM-Med3D-turbo on these 44 datasets to enhance the performance further (we employed our own data split to prevent data leakage, but you can adjust the split to suit your needs). We hope this list can be of assistance to you.
```
AMOS2022
ATM2022
AbdomenCT1K
BTCV_Cervix
BraTS2020
BraTS2021
BrainTumour
Brain_PTM
CAUSE07
CHAOS_Task_4
COSMOS2022
COVID19CTscans
CTPelvic1k
CT_ORG
FLARE21
FLARE22
Heart_Seg_MRI
ISLES_SISS
ISLES_SPES
KiPA22
KiTS
KiTS2021
LAScarQS22_task1
LAScarQS22_task2
LITS
MMWHS
MSD_Colon
MSD_HepaticVessel
MSD_Liver
MSD_Pancreas
MSD_Prostate
MSD_Spleen
PROMISE12
Parse22
Promise09
Prostate_MRI_Segmentation_Dataset
SLIVER07
STACOM_SLAWT
SegThor
Totalsegmentator_dataset
VESSEL2012
VerSe19
VerSe20
WORD
```

**Comment by RayOoooo** [comment]
> Hi, some important training datasets are listed here. We fine-tuned a new version of SAM-Med3D called SAM-Med3D-turbo on these 44 datasets to enhance the performance further (we employed our own data split to prevent data leakage, but you can adjust the split to suit your needs). We hope this list can be of assistance to you.
> 
> ```
> AMOS2022
> ATM2022
> AbdomenCT1K
> BTCV_Cervix
> BraTS2020
> BraTS2021
> BrainTumour
> Brain_PTM
> CAUSE07
> CHAOS_Task_4
> COSMOS2022
> COVID19CTscans
> CTPelvic1k
> CT_ORG
> FLARE21
> FLARE22
> Heart_Seg_MRI
> ISLES_SISS
> ISLES_SPES
> KiPA22
> KiTS
> KiTS2021
> LAScarQS22_task1
> LAScarQS22_task2
> LITS
> MMWHS
> MSD_Colon
> MSD_HepaticVessel
> MSD_Liver
> MSD_Pancreas
> MSD_Prostate
> MSD_Spleen
> PROMISE12
> Parse22
> Promise09
> Prostate_MRI_Segmentation_Dataset
> SLIVER07
> STACOM_SLAWT
> SegThor
> Totalsegmentator_dataset
> VESSEL2012
> VerSe19
> VerSe20
> WORD
> ```

Hi, I notice these datasets is always in SAM-Med3D/utils/data_paths.py. So, what's new in the new version of ckpt? In other words, what are the factors that lead to the huge performance improvement? Just training fine-tuning on fewer datasets to get better results? Or did you fine-tune it for more time, resulting in a noticeable improvement in results? Are there any specific considerations for fine-tuning the dataset selection?

Thank your solid work!
